speech/google/cloud/speech/streaming/container.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Representation of a group of GAPIC Speech API responses."""
+
+from google.cloud.speech.streaming.response import StreamingSpeechResponse
```

Comment:
RE: My suggestion about collapsing into the original package, this import could be `streaming_response` instead of `streaming.response`.

Or you could just define all the streaming classes in one module called `streaming`